phpstan.neon

```diff
@@ -0,0 +1,18 @@
+parameters:
+  level: 3
+  paths:
+    - ./src
+  ignoreErrors:
+    - '#Call to [a-zA-Z0-9\\_:@\(\).]*#'
+    - '#PHPDoc tag @var for variable [a-zA-Z0-9_\$]* contains unknown class [a-zA-Z0-9\\_:@\(\).]*#'
+    - '#PHPDoc tag @throws with type [a-zA-Z0-9\\_:@\(\)|]* is not subtype of Throwable#'
+    - '#Class [a-zA-Z0-9\\_:@\(\)]* not found.#'
+    - '#Parameter [a-zA-Z0-9_\$]* of method [a-zA-Z0-9\\_:@\(\)]* has invalid type [a-zA-Z0-9\\_:@\(\).]*#'
```

Comment:
E.g: `Parameter $config of method Acquia\Console\ContentHub\Command\ContentHubAuditChSettings::syncSettings() has invalid type Drupal\Core\Config\Config.`